src/app/withdraw/components/withdraw-client.tsx

```diff
@@ -188,7 +197,44 @@ const Withdraw = () => {
       [WalletProvider.XVERSE]: callContractXverse,
       [WalletProvider.LEATHER]: callContractLeather,
       [WalletProvider.FORDEFI]: callContractFordefi,
-      [WalletProvider.ASIGNA]: callContractAsigna,
+      [WalletProvider.ASIGNA]: async ({}: {
+        txHex: string;
+        network: StacksNetworkName | StacksNetwork;
+      }) => {
+        const signedTxHex = await openAsignaContractCall(
+          {
+            contractName: opts.contractName!,
+            contractAddress: opts.contractAddress!,
+            functionName: opts.functionName!,
+            functionArgs: [
+              Cl6.uint(satoshiAmount),
+              Cl6.tuple({
+                version: Cl6.bufferFromHex(recipient.type),
+                hashbytes: Cl6.buffer(recipient.hash),
+              }),
+              Cl6.uint(satoshiFee),
+            ],
+            anchorMode: AnchorMode.Any,
+            postConditions: [
+              createFungiblePostCondition(
+                addresses.stacks!.address!,
+                FungibleConditionCode.LessEqual,
+                satoshiAmount + satoshiFee,
+                createAssetInfo(
+                  SBTC_CONTRACT_DEPLOYER,
+                  "sbtc-token",
+                  "sbtc-token",
+                ),
+              ),
+            ],
```

Comment:
nit: should it also include `opts.fee`?

Reply:
for this no, since we're only supporting mainnet